plugins/modules/zos_operator.py

```diff
@@ -161,7 +164,12 @@
 try:
     from zoautil_py import opercmd
 except Exception:
-    opercmd = MissingZOAUImport()
+    opercmd = ZOAUImportError(traceback.format_exc())
+
+try:
```

Comment:
We added module_utils/zoa_version_checker.py so this would behave more consistently

Reply:
Updated 
```
    use_wait_arg = False
    if zoau_version_checker.is_zoau_version_higher_than("1.2.4"):
        use_wait_arg = True
```